packages/hash/api/src/auth/index.ts

```diff
@@ -98,8 +100,19 @@ const setupAuth = (params: {
 
   // eslint-disable-next-line @typescript-eslint/no-misused-promises -- https://github.com/DefinitelyTyped/DefinitelyTyped/issues/50871
   app.use(async (req, _res, next) => {
+    const authHeader = req.header("authorization");
+    const hasAuthHeader = authHeader?.startsWith("Bearer ") ?? false;
+    const sessionToken =
+      hasAuthHeader && typeof authHeader === "string"
+        ? authHeader.slice(7, authHeader.length)
+        : undefined;
+
     const kratosSession = await publicKratosSdk
-      .toSession(undefined, req.header("cookie"))
+      // .toSession({ cookie: req.header("cookie") }
```

Comment:
Was this intended to be left commented out?

Reply:
It was not, thanks :)
f64533fb36bf90408a05902af7870ef375c7fe64